sound/soc/sof/loader.c

```diff
@@ -142,6 +142,18 @@ static int ext_man_get_fw_version(struct snd_sof_dev *sdev,
 	return snd_sof_ipc_valid(sdev);
 }
 
+static int ext_man_get_windows(struct snd_sof_dev *sdev,
+			       struct ext_man_elem_header *hdr)
+{
+	struct sof_ipc_ext_data_hdr *w_ipc;
+	struct ext_man_window *w;
```

Comment:
I think it should be possible to use `const` with all manifest pointers

Reply:
Used functions like `get_ext_windows` or `get_cc_info` doesn't have `const` arguments, so I guess it should be separate PR

Reply:
I thought maybe you could add a patch to this PR to change them to const - before your patches.

Reply:
added